pkg/net/libp2p/libp2p.go

```diff
@@ -476,7 +476,18 @@ func (p *provider) bootstrap(
 		return err
 	}
 
-	bootstrapConfig := bootstrapConfigWithPeers(peerInfos)
+	ownID := p.identity.id
+	filteredPeerInfos := make([]peer.AddrInfo, 0)
+
+	// If the client's own address is mistakenly added to the list of bootstrap
+	// peers, filter it out. This will prevent self-dialing.
```

Comment:
I think it's not a mistake. The list of bootstrap peers is hardcoded in the client.

Reply:
Ok, let's not call it a mistake.
I changed the comment a little in 7dd1bc2d3292e11fbf6294cd557009f342829571.

BTW, I haven't seen the hardcoded peer addresses yet.
I thought you can set this list either via configuration parameter or via ".../_peers/..." file (I think we use the latter in the testnet network).

Reply:
> BTW, I haven't seen the hardcoded peer addresses yet.
I thought you can set this list either via configuration parameter or via ".../_peers/..." file (I think we use the latter in the testnet network).

They are hardcoded for [testnet](https://github.com/keep-network/keep-core/blob/f0b08217226bdefd96455c162fffdb8ef9b9618e/config/_peers/goerli) and [mainnet](https://github.com/keep-network/keep-core/blob/f0b08217226bdefd96455c162fffdb8ef9b9618e/config/_peers/mainnet).